app/src/lib/remote-parsing.ts

```diff
@@ -27,19 +27,19 @@ interface IGitRemoteURL {
 const remoteRegexes: ReadonlyArray<{ protocol: Protocol; regex: RegExp }> = [
   {
     protocol: 'https',
-    regex: new RegExp('^https?://(?:.+@)?(.+)/(.+)/(.+?)(?:/|.git/?)?$'),
+    regex: new RegExp('^https?://(?:.+@)?([^/]+)/([^/]+)/(.+?)(?:/|.git/?)?$'),
```

Comment:
#11035 
Duplicate of #

Reply:
words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com)